bootstrap/status_check_action.go

```diff
@@ -55,6 +55,10 @@ func (a *StatusCheckAction) Run(s *State) error {
 		}
 		if res.StatusCode == 200 {
 			s.StepData[a.ID] = &LogMessage{Msg: "all services healthy"}
+		} else if time.Now().Sub(start) < waitMax {
+			// if services are unhealthy wait out the wait period
+			// before reporting them as unhealthy to the user
+			continue
```

Comment:
The `fail` label is for handling not being able  to communicate with the status service.
In this case we got a response but is was non-200, I had just forgotten to add the sleep so it was actually a busy loop, oops.